docs/the-git-based-process-you-need-for-mainframe-development-DRAFT/pipeline-design-and-implementation-supporting-the-workflows.md

```diff
@@ -17,45 +17,45 @@
 1. [Configurations to support working with Feature branches](#configurations-to-support-working-with-feature-branches)
 
     * [User Build setup](#user-build-setup)
-  
+
     * [Pipeline build of feature branches](#pipeline-build-of-feature-branches)
-  
+
     * [Package and Deploy a feature for testing in controlled test environments](#package-and-deploy-a-feature-for-testing-in-controlled-test-environments)
-  
+
 2. [House keeping recommendations](#house-keeping-recommendations)
 
 3. [The Build pipeline for main, epic and release branches](#the-build-pipeline-for-main-epic-and-release-branches)
 
     * [Build pipeline: Build and Test step](#build-pipeline-build-and-test-step)
-  
+
     * [Build pipeline: Install outputs to a shared DEV-TEST runtime](#build-pipeline-install-outputs-to-a-shared-dev-test-runtime)
-  
+
     * [Build pipeline: Analyze step](#build-pipeline-analyze-step)
-  
+
 4. [The Release pipeline: Build, package and deploy](#the-release-pipeline-build-package-and-deploy)
 
     * [Release pipeline: Build stage](#release-pipeline-build-stage)
-  
+
     * [Release pipeline: Packaging stage](#release-pipeline-packaging-stage)
-  
+
     * [Release pipeline: Deployment stage](#release-pipeline-deployment-stage)
-  
+
 5. [Deployment to production](#deployment-to-production)
 
 [Conclusion](conclusion.md#conclusion)
 
 [Appendix](appendix.md#appendix)
 
 
-# Pipeline design and implementation supporting the workflows 
+# Pipeline design and implementation supporting the workflows
 
 In this section, the technical implementation of the different composition of pipeline of the outlined development methodology will be detailed. If a workflow demands a specific configuration, it will be covered within the same section.
 
 ## Configurations to support working with Feature branches
 
 ![](images/media/image16.png) When the developers start working on a new task, they will first create a feature branch. Feature branches are created off the latest code state of the state of the source configuration -- the *main*, *epic* or *release* maintenance branch.
 
-If the feature branch was created on the central Git repository, the developers can use the IDE, a terminal, or another Git interface on their local workstation to clone or fetch the new feature branch from the central Git repository down to their local working tree in which the changes will be implemented.
+If the feature branch was created on the central Git repository, the developers can use the IDE, a terminal, or another Git interface on their local workstation to clone or pull the new feature branch from the central Git repository. They check out the feature branch in which the changes will be implemented.
```

Comment:
> They check out the feature branch...

Thanks @dennis-behm for taking my suggestion! But did we want to go with, "They switch to the feature branch..." per the discussion in our [comment thread](https://github.com/IBM/dbb-zappbuild/pull/420#discussion_r1341315213)?

Reply:
Absolutely. Missed to fix this one.